config/default/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "medic-cht-ref",
+  "version": "0.1.0",
+  "description": "",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "eslint": "eslint *.json test/**/*.js",
+    "test": "npm run eslint && npm run test-unit",
+    "test-forms": "npm run eslint && mocha --reporter spec test/forms/**/*.spec.js --timeout 10000",
+    "test-tasks": "npm run eslint && mocha --reporter progress test/tasks/*.spec.js --timeout 10000",
+    "test-targets": "npm run eslint && mocha --reporter progress test/targets/*.spec.js --timeout 10000",
+    "test-contact-summary": "npm run eslint && mocha --reporter progress test/contact-summary/*.spec.js --timeout 10000",
+    "test-unit": "mocha --reporter spec test/**/*.spec.js --timeout 20000",
+    "travis": "npm test"
+  },
+  "devDependencies": {
+    "@medic/contact-summary-testing": "^1.1.4",
```

Comment:
Is this `@medic/contact-summary-testing` dependency required?

Reply:
No, it's not required. Removed it.